mk/README

```diff
@@ -81,7 +78,8 @@ genlink
 
   This module adds an support for the user to the linker script generator. The
```

Comment:
while we're haxing this file... this sentence doesn't make a whole lot of sense. This module adds support for what ?